mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -211,8 +217,9 @@ func resourceBigtableInstanceCreate(d *schema.ResourceData, meta interface{}) er
 
 	defer c.Close()
 
-	err = c.CreateInstanceWithClusters(ctx, conf)
-	if err != nil {
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, d.Timeout(schema.TimeoutCreate)-5*time.Second)
```

Comment:
What is the function of the "-5" here? I'm not seeing any other logic around the codebase use a similar construct

Reply:
Fixed